pipeline_reviews/extract.py

```diff
@@ -19,9 +19,12 @@ def __init__(self, message="No new games were found in the last 2 weeks!"):
 
 def get_number_of_reviews(game_id: int) -> dict:
     """Retrieves information about all reviews from a given game ID"""
-    request = requests.get(f"https://store.steampowered.com/appreviews/{game_id}?json=1")
-    reviews_info = request.json()
-    return reviews_info["query_summary"]["total_reviews"]
+    try:
+        request = requests.get(f"https://store.steampowered.com/appreviews/{game_id}?json=1", timeout=10)
+        reviews_info = request.json()
+        return reviews_info["query_summary"]["total_reviews"]
+    except requests.exceptions.Timeout:
+        return 0
```

Comment:
You return number of reviews here, which is int, but typehint says dict

Reply:
Good catch!! This was previously a dict with other values too but yeah the function name and functionalities were changed. Will change!